### DISCUSSION TITLE
沒有電腦，使用平板或手機也可以操作Jupiter notebook

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#183] [General]
先連到這邊：https://jupyter.org/try

然後選擇想要操作的環境
![image](https://user-images.githubusercontent.com/1768891/130808464-a8ee2707-f697-4e4c-afba-0b1fb958b229.jpeg)

然後就會被redirect到Jupiter notebook或Jupiter lab的網頁了。

如此一來，使用平板，沒有電腦也可以操作Jupiter notebook，隨時隨地可以寫python程式。

喔對，google colab也可以做到相同的事情。也可以搜尋“colab”，然後玩玩看吧～

**Comment by enoch-lightning** [comment]
簡直太**方便**了！這樣也是使用手機與自己的電腦跑程式嗎？還是是使用別人的電腦？

**Reply by chia-kai-chang** [reply]
這樣算用別人的電腦喔

**Reply by enoch-lightning** [reply]
收到。謝謝老師。原來這樣算是用別人的電腦。

**Comment by xCk27x** [comment]
剛剛試了一下 jupyter lab 跟 notebook 都能順利運作 
雖然要一直切換輸入鍵盤很麻煩 但能直接用手機跑python真的很方便

**Reply by chia-kai-chang** [reply]
沒錯，真是ipad平板使用者的好工具。

**Comment by enoch-lightning** [comment]
我發現常常無法使用，不知道是怎麼回事。
